packages/core/content-manager/admin/src/components/Widgets.tsx

```diff
@@ -301,10 +301,12 @@ const DonutChartSVG = ({ data }: { data: ChartData[] }) => {
         <Portal
           style={{
             position: 'fixed',
-            left: tooltip.x + 16,
-            top: tooltip.y - 16,
+            left: tooltip.x,
+            top: tooltip.y,
             zIndex: 2,
+            transform: 'translate(1.6rem, 1.6rem)',
           }}
```

Comment:
my point was that the _dynamic_ part (tooltip.x and tooltip.y) should be the one in the transform, because it's the transform that super cheap for the browser to render when it updates